### DISCUSSION TITLE
LiveVideoBroadcaster

### DISCUSSION THREAD

**Discussion by gaelson** [ant-media/Ant-Media-Server#5452] [General]
Hello,
I try the [LiveVideoBroadcaster](https://github.com/ant-media/LiveVideoBroadcaster) android SDK.  I don't get the good result. I get some error in Android studio.
Some body can Tell me if this sdk is update and works very well?please

Regards

**Comment by yashtandon113** [comment]
@gaelson 
Since Ant Media no longer maintains the repository, it is very old. If anyone wants to use and alter it, it is available as open source to the public. We advise users to use encoders like OBS, VMIX, etc. for RTMP. We do not provide an RTMP SDK.
​
Please refer to below link for RTMP publishing to Ant media server.
https://antmedia.io/docs/category/rtmp/

You may also use some third party mobile applications for RTMP.

Please let us know if you have any questions.

**Reply by mekya** [reply]
Thank you for the answer @yashtandon113, you're right. 

@gaelson
I've something to add, it's likely about upgrading the audio studio or gradle plugins may help resolve the problem.
If you fix the problem, please send a PR to the repo.  

Regards
Oguz

**Comment by gaelson** [comment]
Hello, thank you all... 
I wanna know if ant media server doesn't allow now playing rtmp url like VLC? 
I create a live streaming, I can publish on the server but I don't find the way to get abck rtmp url to play it unless hls url.

**Reply by yashtandon113** [reply]
@gaelson 
Ant Media Server does not recommend RTMP stream playback because it is no longer maintained. Instead, you can use WebRTC (0.5 seconds), Dash (3-5 sec) or HLS (8-10 sec) playback options. By default it is disabled but if you would like to use it, please check the following GitHub discussion: https://github.com/orgs/ant-media/discussions/4874
